test/likelihoodratiotest.jl

```diff
@@ -1,46 +1,50 @@
-using MixedModels, Test
-import GLM: ProbitLink
+using MixedModels
+using Test
+
+using MixedModels: dataset, likelihoodratiotest
+using GLM: ProbitLink
 
 @testset "likelihoodratio test" begin
-    slp = MixedModels.dataset(:sleepstudy);
+    slp = dataset(:sleepstudy);
     fm0 = fit(MixedModel,@formula(reaction ~ 1 + (1+days|subj)),slp);
     fm1 = fit(MixedModel,@formula(reaction ~ 1 + days + (1+days|subj)),slp);
-    lrt = MixedModels.likelihoodratiotest(fm0,fm1);
+    lrt = likelihoodratiotest(fm0,fm1);
 
     @test [deviance(fm0), deviance(fm1)] == lrt.deviance
-    @test deviance(fm0) - deviance(fm1) == first(lrt.tests.deviancediff)
-    @test first(lrt.tests.dofdiff) == 1
+    @test deviance(fm0) - deviance(fm1) == only(lrt.tests.deviancediff)
+    @test only(lrt.tests.dofdiff) == 1
     @test sum(map(length,lrt.tests)) == 3
     @test sum(map(length,lrt.pvalues)) == 1
     @test sum(map(length,lrt.models)) == 4
     @test length(lrt.formulae) == 2
     show(IOBuffer(),lrt);
+    @test :pvalues in propertynames(lrt)
 
 
     # mix of REML and ML
     fm0 = fit(MixedModel,@formula(reaction ~ 1 + (1+days|subj)),slp, REML=true);
-    @test_throws ArgumentError MixedModels.likelihoodratiotest(fm0,fm1)
+    @test_throws ArgumentError likelihoodratiotest(fm0,fm1)
 
     # differing FE with REML
     fm1 = fit(MixedModel,@formula(reaction ~ 1 + days + (1+days|subj)),slp, REML=true);
-    fm10 = fit(MixedModel,@formula(reaction ~ 1 + days + (1|subj)),slp, REML=true);
-    @test_throws ArgumentError MixedModels.likelihoodratiotest(fm0,fm1);
+    fm0 = fit(MixedModel,@formula(reaction ~ 1 + days + (1|subj)),slp, REML=true);
```

Comment:
I think the test using `fm10` was removed (maybe even before this made it into `master`). `fm0` shouldn't be redefined -- because `fm0` and `fm1` need to have different fixed effects.

```suggestion
 
```